appveyor.yml

```diff
@@ -48,9 +50,31 @@ environment:
     -
       PUPPET_GEM_VERSION: ~> 6.0
       RUBY_VERSION: 25-x64
-      CHECK: spec
-matrix:
-  fast_finish: true
+      CHECK: spec     
+    -
+      RUBY_VERSION: 25-x64
+      ACCEPTANCE: yes
+      TARGET_HOST: localhost
+    -
+      RUBY_VERSION: 25-x64
+      ACCEPTANCE: yes
+      TARGET_HOST: localhost
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+  #matrix:
+  #  fast_finish: true
+for:
+-
+  matrix:
+    only:
+      - ACCEPTANCE: yes
+  install:
+    - set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+    - bundle install --jobs 4 --retry 2 --without build
+    - type Gemfile.lock
+  test_script:
+    - ps: Set-Content -Value 'modulepath = C:/projects/puppetlabs-dsc/spec/fixtures/modules' -Path 'C:\ProgramData\PuppetLabs\code\environments\production\environment.conf'
```

Comment:
Hey @tphoney I was able to get the module path in correctly by adding this (and line 11 above), but the powershell call that results from this line https://github.com/ThoughtCrhyme/puppetlabs-dsc/blob/FM-8072/spec/lib/dsc_utils_litmus.rb#L11 ends up with a newline character in it for some reason. See here: https://ci.appveyor.com/project/puppetlabs/puppetlabs-dsc/builds/25433595/job/m2oyl1ojldn5sgpl#L1230

I've tried stripping it out before forming the call, but it didn't work: https://github.com/puppetlabs/puppetlabs-dsc/pull/414/files#diff-58d69cbefccdc435228cb60aaf6b8552R6
and
https://github.com/puppetlabs/puppetlabs-dsc/pull/414/files#diff-58d69cbefccdc435228cb60aaf6b8552R10

Reply:
Ok, the stripping out of the newline now works, but now I'm getting "Test-Path was not expected at this time" https://ci.appveyor.com/project/puppetlabs/puppetlabs-dsc/builds/25459972/job/1id22a25pbnf30a7#L1231

First thought is this is a problem in run_shell, haven't debugged through it though, I'll check on Monday.